Resources/Private/Fusion/Document/Fragment/StyleSheets.fusion

```diff
@@ -3,8 +3,8 @@
  */
 prototype(Neos.Demo:Document.Fragment.StyleSheets) < prototype(Neos.Fusion:Component) {
     renderer = afx`
+        <link href="https://fonts.googleapis.com/css?family=Work+Sans:400&display=swap" rel="stylesheet" />
```

Comment:
Can we prevent having to load a font from Google?

Reply:
We could also include these files, yes. But in my opinion, we can make the package with this change a bit smaller…

Reply:
Done